specifications/api/swagger.yaml

```diff
@@ -723,6 +705,35 @@ definitions:
       outputs: *transactionOutputs
       status: *transactionStatus
 
+  ApiWalletDelegation: &ApiWalletDelegation
+    description: Delegation settings
+    type: object
+    required:
+      - active
+    properties:
+      active:
+        type: object
+        <<: *ApiDelegationStatus
+        description: Currently active delegation status.
+        example:
+          status: delegating
+          target: 1423856bc91c49e928f6f30f4e8d665d53eb4ab6028bd0ac971809d514c92db1
+      next:
+        type: object
+        <<: *ApiDelegationStatus
+        description: Next delegation settings about to change. Only present when a status change has to be reported.
```

Comment:
Maybe
```
Next delegation status that is not yet in effect. Only present when a new delegation certificate has been submitted or discovered.
```
?

- `delegation settings` -> `delegation status` for consistency with ApiDelegationStatus above.
- `has to be` -> `has been`

Reply:
changed to : 
```
Next delegation status that is not yet in effect. Only present when a new delegation certificate has been discovered.
```
it has to be discovered. submitting does not affect this field